python/pyiceberg/catalog/dynamodb.py

```diff
@@ -0,0 +1,776 @@
+
+    current_timestamp_ms = str(round(time() * 1000))
+    _dict = {
+        DYNAMODB_COL_IDENTIFIER: {
+            "S": f"{database_name}.{table_name}",
+        },
+        DYNAMODB_COL_NAMESPACE: {
+            "S": database_name,
+        },
+        DYNAMODB_COL_VERSION: {
+            "S": str(uuid.uuid4()),
+        },
+        DYNAMODB_COL_CREATED_AT: {
+            "N": current_timestamp_ms,
+        },
+        DYNAMODB_COL_UPDATED_AT: {
+            "N": current_timestamp_ms,
+        },
+    }
+
+    for key, val in properties.items():
+        _dict[_add_property_prefix(key)] = {"S": val}
+
+    _dict[_add_property_prefix(TABLE_TYPE)] = {"S": ICEBERG.upper()}
+    _dict[_add_property_prefix(METADATA_LOCATION)] = {"S": metadata_location}
+    _dict[_add_property_prefix(PREVIOUS_METADATA_LOCATION)] = {"S": ""}
+
+    return _dict
+
+
+def _get_rename_table_item(from_dynamo_table_item: Dict[str, Any], to_database_name: str, to_table_name: str) -> Dict[str, Any]:
+    _dict = from_dynamo_table_item
+    current_timestamp_ms = str(round(time() * 1000))
+    _dict[DYNAMODB_COL_IDENTIFIER]["S"] = f"{to_database_name}.{to_table_name}"
+    _dict[DYNAMODB_COL_NAMESPACE]["S"] = to_database_name
+    _dict[DYNAMODB_COL_VERSION]["S"] = str(uuid.uuid4())
+    _dict[DYNAMODB_COL_UPDATED_AT]["N"] = current_timestamp_ms
+    return _dict
+
+
+def _get_create_database_item(database_name: str, properties: Properties) -> Dict[str, Any]:
+    current_timestamp_ms = str(round(time() * 1000))
+    _dict = {
+        DYNAMODB_COL_IDENTIFIER: {
+            "S": DYNAMODB_NAMESPACE,
+        },
+        DYNAMODB_COL_NAMESPACE: {
+            "S": database_name,
+        },
+        DYNAMODB_COL_VERSION: {
+            "S": str(uuid.uuid4()),
+        },
+        DYNAMODB_COL_CREATED_AT: {
+            "N": current_timestamp_ms,
+        },
+        DYNAMODB_COL_UPDATED_AT: {
+            "N": current_timestamp_ms,
+        },
+    }
+
+    for key, val in properties.items():
+        _dict[_add_property_prefix(key)] = {"S": val}
+
+    return _dict
+
+
+def _get_update_database_item(namespace_item: Dict[str, Any], updated_properties: Properties) -> Dict[str, Any]:
+    current_timestamp_ms = str(round(time() * 1000))
+
+    # noinspection PyDictCreation
+    _dict = {}
+    _dict[DYNAMODB_COL_IDENTIFIER] = namespace_item[DYNAMODB_COL_IDENTIFIER]
+    _dict[DYNAMODB_COL_NAMESPACE] = namespace_item[DYNAMODB_COL_NAMESPACE]
+    _dict[DYNAMODB_COL_VERSION] = {
+        "S": str(uuid.uuid4()),
+    }
+    _dict[DYNAMODB_COL_CREATED_AT] = namespace_item[DYNAMODB_COL_CREATED_AT]
+    _dict[DYNAMODB_COL_UPDATED_AT] = {
+        "N": current_timestamp_ms,
+    }
+
+    for key, val in updated_properties.items():
+        _dict[_add_property_prefix(key)] = {"S": val}
+
+    return _dict
+
+
+def _get_create_catalog_attribute_definitions() -> List[Dict[str, str]]:
```

Comment:
instead of methods, we could make these also constants.

Reply:
Done.